.appveyor.yml

```diff
@@ -59,28 +68,30 @@ install:
   - ps: Set-Location -Path $env:APPVEYOR_BUILD_FOLDER
 
   # Verify CWD
-  - ps: Write-Host "[+] Current working directory is ..."
+  - ps: Write-Host "[+] Current PowerShell directory is ..."
   - ps: Get-Location
   - bash -e -l -c "cd $APPVEYOR_BUILD_FOLDER && ls -alh"
 
-  # Install Python modules
-  - ps: python -m pip install --upgrade pip
-  # - ps: python -m pip install --upgrade packager
-  - ps: python -m pip install --upgrade pyinstaller
-
   # Install NPM modules
-  # - bash -e -l -c "cd /C/Tridactyl && npm install -g windows-build-tools"
   - bash -e -l -c "cd $APPVEYOR_BUILD_FOLDER && npm install"
 
 build_script:
   # Add Python-3.6 to %PATH%
   - ps: $env:PATH = "C:\Python36-x64\Scripts;$env:PATH"
   - ps: $env:PATH = "C:\Python36-x64;$env:PATH"
-  - ps: Copy-Item -Path "C:\Python36-x64\Python.exe" -Destination "C:\Python36-x64\Python3.exe"
+  - ps: >-
+        Copy-Item `
+        -Path "C:\Python36-x64\Python.exe" `
+        -Destination "C:\Python36-x64\Python3.exe"
+  - python --version
+
+  # Install Python modules under Python 3.6
+  - ps: python -m pip install --upgrade pip
+  - ps: python -m pip install --upgrade pyinstaller
```

Comment:
As I mentioned in #620, this won't be used for anything unless we add pyinstaller to build.sh.

Reply:
(these two comments apply to npm test too, obviously :) );

Reply:
Well, I don't think it'll hurt anything. But I'm okay to comment them out for now if you want. Let me know.

BTW, did you want to have the `build.sh` from #560 here under #625? As said below, I think building EXE on Windows would be the right approach long term.

Thanks.

Reply:
No, as a separate PR, please :)